application/controllers/DashboardsController.php

```diff
@@ -0,0 +1,483 @@
+            }
+
+            $home->setActive()->loadDashboardEntries();
+            foreach ($value as $pane => $indexOrValues) {
+                if (! $home->hasEntry($pane) && (! $orgHome || ! $orgHome->hasEntry($pane))) {
+                    Notification::error(sprintf(t('Dashboard pane "%s" not found'), $pane));
+                    break;
+                }
+
+                /** @var Pane $pane */
+                $pane = $home->hasEntry($pane) ? $home->getEntry($pane) : $orgHome->getEntry($pane);
+                if (! is_array($indexOrValues)) {
+                    if ($orgHome && $orgHome->hasEntry($pane->getName()) && $home->hasEntry($pane->getName())) {
+                        Notification::error(sprintf(
+                            t('Dashboard "%s" already exists within "%s" home'),
+                            $pane->getTitle(),
+                            $home->getTitle()
+                        ));
+                        break;
+                    }
+
+                    // Perform DB updates
+                    $home->reorderWidget($pane, (int) $indexOrValues, $orgHome);
+                    if ($orgHome) {
+                        // In order to properly update the dashlets id (user + home + pane + dashlet)
+                        $pane->manageEntry($pane->getEntries());
+                    }
+
+                    Notification::success(sprintf(
+                        t('%s dashboard pane "%s" successfully'),
+                        $orgHome ? 'Moved' : 'Updated',
+                        $pane->getTitle()
+                    ));
+                    break;
+                }
+
+                foreach ($indexOrValues as $dashlet => $index) {
+                    if (! $pane->hasEntry($dashlet) && (! $orgPane || ! $orgPane->hasEntry($dashlet))) {
+                        Notification::error(sprintf(t('Dashlet "%s" not found'), $dashlet));
+                        break;
+                    }
+
+                    if ($orgPane && $orgPane->hasEntry($dashlet) && $pane->hasEntry($dashlet)) {
+                        Notification::error(sprintf(
+                            t('Dashlet "%s" already exists within "%s" dashboard pane'),
+                            $dashlet,
+                            $pane->getTitle()
+                        ));
+                        break;
+                    }
+
+                    $dashlet = $pane->hasEntry($dashlet) ? $pane->getEntry($dashlet) : $orgPane->getEntry($dashlet);
+                    $pane->reorderWidget($dashlet, (int) $index, $orgPane);
+
+                    Notification::success(sprintf(
+                        t('%s dashlet "%s" successfully'),
+                        $orgPane ? 'Moved' : 'Updated',
+                        $dashlet->getTitle()
+                    ));
+                }
+            }
+        }
+
+        if ($widgetType !== 'Dashlets' || ($orgHome && $orgPane)) {
+            $this->redirectNow($reroutePath);
```

Comment:
Replace this with this:

```suggestion
            $this->createTabs();
            $this->dashboard->activate('dashboard_settings');
            $this->sendMultipartUpdate();
```

This will only update the tabs. The dummy content and the header below can also be removed then.
Make also sure to always set `addToHistory=false` and `scripted=true` when calling `loadUrl()` in your JS.